packages/liveblocks-core/src/client.ts

```diff
@@ -124,7 +124,7 @@ export function createClient(options: ClientOptions): Client {
 
   const rooms = new Map<
     string,
-    InternalRoom<JsonObject, LsonObject, BaseUserMeta, Json>
+    Room<JsonObject, LsonObject, BaseUserMeta, Json>
```

Comment:
This structure now directly holds and returns `Room` instances. Private methods (like `.connect()` and `.disconnect()`) that are needed for the internal implementation are no longer sent in an enclosing wrapper type (e.g. `InternalRoom`), but now just live on `room.__internal.privateMethod()`, like some other internal methods that already lived there. These methods are not supposed to be called from the outside, and they will be invisible to TypeScript users.